specification/loadtestservice/LoadTestService/client.tsp

```diff
@@ -33,7 +34,7 @@ interface AdministrationOperations {
 })
 interface TestRunOperations {
     // TODO check with service team why is Python different here?
-    @projectedName("python", "TestRun")
+    @projectedName("python", "BeginTestRun")
```

Comment:
Same question as the above, if there's a difference in how the operation was defined then we'll need to investigate what changed.

Reply:
same with https://github.com/Azure/azure-rest-api-specs/pull/24801#discussion_r1263217765.